documentation/source/users/rmg/liquids.rst

```diff
@@ -162,34 +162,25 @@ where `A, B, E, S` and `L` are the same solute descriptors used in the Abraham m
 :math:`\Delta G_{\rm solv}^{*}`. The lowercase coefficients `c', a', b', e', s'` and `l'` depend only on the solvent and were obtained
 by fitting to experimental data.
 
-The solvent descriptors (`c, e, s, a, b, l, c', a', b', e', s', l'`) are largely treated as regressed empirical coefficients.
-Parameters are provided in RMG's database for the following solvents:
+RMG's database has a solute library that contains the solute descriptors of around 300 compounds and it is located at
+``RMG-database/input/solvation/libraries/solute.py`` in RMG-database. For the compounds that are not found in the
+solute library, the group additivity method is used to estimate the solute descriptors as described below under
+`Group additivity method for solute descriptor estimation`.
 
-#. acetonitrile
-#. benzene
-#. butanol
-#. carbontet
-#. chloroform
-#. cyclohexane
-#. decane
-#. dibutylether
-#. dichloroethane
-#. dimethylformamide
-#. dimethylsulfoxide
-#. dodecane
-#. ethanol
-#. ethylacetate
-#. heptane
-#. hexadecane
-#. hexane
-#. isooctane
-#. nonane
-#. octane
-#. octanol
-#. pentane
-#. toluene
-#. undecane
-#. water
+The solvent descriptors (`c, e, s, a, b, l, c', a', b', e', s', l'`) are largely treated as regressed empirical coefficients.
+The solvent descriptors are fitted using in-house experimental solute parameter, solvation free energy, and solvation
```

Comment:
I'm confused, did you run experiment to generate all of the data used in that paper?

Reply:
it's fixed in this fixup commit 2d234db.